Fixes #181: User controller has static username

mfavetti opened the pull request.

dfranco commented:
Good catch @mfavetti !

I tought I've removed all hardcoded values while working on latest release.

Only one issue, could ensure you're using conventional commit please ?
https://www.conventionalcommits.org/en/v1.0.0/

As I'm using [release please](https://github.com/googleapis/release-please) to generate GitHub release, I'm afraid the changlog will not be generate properly with your commit message

You can have a look at https://github.com/googleapis/release-please#how-should-i-write-my-commits, and I'll will update a bit the contributor guide in the documentation.

Thanks

mfavetti commented:
> Good catch @mfavetti !
> 
> I tought I've removed all hardcoded values while working on latest release.
> 
> Only one issue, could ensure you're using conventional commit please ? https://www.conventionalcommits.org/en/v1.0.0/
> 
> As I'm using [release please](https://github.com/googleapis/release-please) to generate GitHub release, I'm afraid the changlog will not be generate properly with your commit message
> 
> You can have a look at https://github.com/googleapis/release-please#how-should-i-write-my-commits, and I'll will update a bit the contributor guide in the documentation.
> 
> Thanks

ok, updated the commit

dfranco commented:
Thanks
